fr_fr.lang

```diff
@@ -120,7 +119,7 @@ gui.xaero_preset_topright=En Haut À Droite
 gui.xaero_waypoint_teleport=Téléporter
 gui.xaero_waypoints_key=Points De Navigation
 gui.xaero_display_coords=Voir Coordonnées
-gui.xaero_disable_enable=Activer/Désactiver
+gui.xaero_disable_enable=Désactiver/Activer
```

Comment:
What do you think about splitting this into two separate lines to reduce confusion among translators?

As you can see here, some of them have made a harmless-looking change, and now as a consequence the button on the waypoint screen works incorrectly from the user's perspective.

Reply:
I agree that it would be better. I haven't changed this because it requires doing the same for every file, if I want to keep the translations. I can see how you would be able to automate it though, so we'll see.

Reply:
I can help you with it, of course. Just edit the source U.S. English translation and I will take care of all the others. The only downside is that I don't visit this repo on GitHub every day, so I may notice with some delay.

Reply:
I can help you with this, of course. Just edit the source U.S. English translation and I will take care of all the others. The only downside is that I don't visit this repo on GitHub every day, so I may notice with some delay.

Reply:
I appreciate your help! No problem, I should pay more attention to this repository myself.